cnf-tests/Dockerfile

```diff
@@ -4,7 +4,7 @@ COPY . .
 RUN make test-bin
 RUN git rev-list -1 HEAD > ./cnf-tests/bin/cnftests-sha.txt
 
-FROM quay.io/openshift/origin-oc-rpms:4.8 AS oc
+FROM quay.io/openshift/origin-oc-rpms:4.13 AS oc
```

Comment:
for the next time: bumping these images is best done in separate PR (or we should update our prow config to consume the current PR, which is probably a bigger undertaking)